src/components/popover/popover-trigger/popover-trigger.stories.ts

```diff
@@ -79,7 +79,6 @@ const popover = (): TemplateResult => html`
       Some content.
       <sbb-link
         size="s"
-        variant="block"
```

Comment:
I'd vote to using block variant again. Because inline probably destabilizes chromatic builds again as the total width gets too large (just detected, I am sorry).